docs/sphinx/quickstart.rst

```diff
@@ -146,14 +146,14 @@ the distributed aspects of |hpx|.
       ... examples.quickstart.hello_world_distributed
       ... hello_world_distributed
 
-   It is also possible to build e.g. all quickstart examples using ``make
+   It is also possible to build all quickstart examples using ``make
```

Comment:
I think the `e.g.` serves a purpose here. The intention is, to show that one can build only a subset, not just everything (aka the `all` target). There are many more targets which can be displayed with `make help`, or `ninja -t targets`

Reply:
@sithhell The e.g. doesn't flow well grammatically speaking, so I'll look into rewording the sentence to sound better but preserve the meaning. Thanks!